test/metabase/driver/common/parameters/values_test.clj

```diff
@@ -29,10 +29,15 @@
            (#'values/value-for-tag
             {:name "id", :display-name "ID", :type :text, :required true, :default "100"} nil)))))
 
+(defn- to-field [{:keys [base_type] :as field}]
+  (map->FieldInstance
+   (merge {:coercion_strategy nil, :effective_type base_type, :semantic_type nil}
+          field)))
```

Comment:
You could use `mt/derecordize` instead to make FieldInstances regular maps for test purposes

Reply:
There's also `mt/object-defaults` that add default values for keys for an object, so we don't have to repeat things like this all over the code or go update a bunch of tests when we add new columns. I'd use that instead